initfiles/componentfiles/configxml/dali.xsd

```diff
@@ -409,6 +409,13 @@
         </xs:appinfo>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="allowedClockVariance" use="optional" type="xs:string" default="5">
+      <xs:annotation>
+        <xs:appinfo>
+          <tooltip>Maximum number of seconds that client clocks can vary from Dali clock, used when checking permissions request digital signature.</tooltip>
```

Comment:
Should we note that this is used when clocks can't be synchronized for whatever reason?

Reply:
I don't feel the need to discuss the reason for the variance, only that we can account for it.

Reply:
ok, not a big deal.